.travis.yml

```diff
@@ -6,7 +6,7 @@ after_success:
   - npm run codecov
 before_deploy:
   - cd docs
-  - npm install
+  - npm ci
```

Comment:
This edit uses [`npm ci`](https://docs.npmjs.com/cli/ci) as the linked npm doc suggests:
> Configure Travis to build using npm ci instead of npm install

###### Benefits
>  It can be significantly faster than a regular npm install by skipping certain user-oriented features. It is also more strict than a regular install, which can help catch errors or inconsistencies caused by the incrementally-installed local environments of most npm users.

Reply:
👍 this used to have issues with npm versions on Travis, but looks like this might work now